### ISSUE TITLE
[Feature request] MM3D """Difficulty Fixes""" that remain unchanged in the mod.

### ISSUE THREAD

**Issue by Nevik9191** [leoetlino/project-restoration#153]
Hi i found a large quantity of "difficulty changes" made to MM3D that haven't been changed or requested as of yet in this mod.
As such I wanted to post a list of a few of the more odd ones that I feel are notable for your consideration of change. As well as link to a full list of changes made within MM3D for reference.

The maze of fire in the Deku Shrine was switched to metal gates that don't hurt the player to make the game easier.

The eggs in the Pirate Fortress fish tank that were guarded by Shell Blades no longer have Shell Blades guarding them.

The great fairy of power and wisdom along with their rewards have swapped places for some reason.

The Gibdos who asked for 10 bombs, 5 magic beans, and 10 Deku nuts now only want one of each item. Additionally the Gibdo who wants a blue potion now accepts a red potion as well.

Powder kegs now cost 20 rupees instead of 100. (However this is probably a good thing that doesn't need to be changed.)

If you are interested and I'm sure you probably already know about this a full list can be found here. https://zelda.gamepedia.com/The_Legend_of_Zelda:_Majora%27s_Mask_3D

I know you are really busy so thank you for all of your hard work so far. I'm sorry if you have had any of these mentioned before I tried to check all closed issues first.

**Comment by Luigimeansme** [comment]
These are all completely unnecessary changes. The fire was changed to metal grates so you won't get sent back to the start when playing as the Deku. The shell blades were removed because it was annoying to fight them in a tight space. The fairy rewards were changed so you could get double magic before getting the Goron, because the Goron roll uses up magic very quickly. The Gibdos were changed to make it less obnoxious to gather up all the items you needed, like making it so you don't have to sit there spamming A in front of a business scrub to buy a bunch of beans.

**Comment by CalmWeaver27** [comment]
I agree with most of the difficulty adjustments because they do feel like genuine positive changes that no longer waste the players time and takes away a lot of needless frustration. I will agree that I think the original placement of the Great Fairy powerups was more ideal, though. Getting the Large Sword Spin early on feels better since you are still doing battle with Normal Link for much longer during that period and you don't quite need the Large MP Bar just yet. Getting the doubled MP Bar that early makes the smaller MP Bar, and MP management as a whole, feel almost pointless. Not only that, but Goron Link *is* manageable with the Small MP Bar, and getting the double sized one after Snowhead Temple, in which you start to feel the need for improved MP management, makes for a much more satisfying reward for collecting the Stray Fairies in said dungeon.

All in all, the original Great Fairy placement made for a far more satisfying game flow. If swapping them is doable, I'd recommend it. The rest of the difficulty changes are fine as they are, while this one would be a nice return, and would bring back what was originally a more rewarding experience, in my opinion.
